arbitrum-docs/node-running/reference/software-releases.md

```diff
@@ -0,0 +1,56 @@
+---
+title: 'ArbOS software releases'
+sidebar_label: 'ArbOS software releases'
+author: dlee
+---
+
+:::info
+This page is meant to give a high level summary of node software releases, with particular emphasis on releases that bump the ArbOS version. To find information on all releases of Nitro, including the latest release, and accompanying changelogs, please visit the [Nitro Github Repository](https://github.com/OffchainLabs/nitro/releases).
+
+:::
+
+The name of the node software that is used to power Arbitrum is called Nitro. While Nitro releases can get shipped as frequently as multiple times per week, there are a few Nitro releases that carry an upgrade to [ArbOS](https://forum.arbitrum.foundation/t/arbitrum-arbos-upgrades/19695) - the Layer 2 EVM hypervisor that facilitiates the execution environment of an Arbitrum chain. Releases of ArbOS can be considered as Arbitrum's equivalent to a hard fork and are special because they could alter a node's ability to produce valid Arbitrum blocks and are therefore **mandatory to pick up** as a validator of Arbitrum One and Nova. For Arbitrum One and Nova, ArbOS version upgrades require that the Arbitrum DAO pass a governance proposal. [This is an example of a Nitro release that contains an ArbOS version bump, specifically to ArbOS 11](https://github.com/OffchainLabs/nitro/releases/tag/v2.2.0).
```

Comment:
```suggestion
The name of the node software that is used to power Arbitrum is called Nitro. While Nitro releases can get shipped as frequently as multiple times per week, there are a few Nitro releases that carry an upgrade to [ArbOS](https://forum.arbitrum.foundation/t/arbitrum-arbos-upgrades/19695) - the Layer 2 EVM hypervisor that facilitiates the execution environment of an Arbitrum chain. Releases of ArbOS can be considered as Arbitrum's equivalent to a hard fork and are special because they could alter a node's ability to produce valid Arbitrum blocks and are therefore **mandatory to pick up** as a validator of a public Arbitrum chain. ArbOS upgrades are carried out by the chain's owner; in the case of Arbitrum One and Nova, the owner is the Arbitrum DAO and so an upgrade will require a governance proposal and vote to pass to complete the upgrade. [This is an example of a Nitro release that contains an ArbOS version bump, specifically to ArbOS 11](https://github.com/OffchainLabs/nitro/releases/tag/v2.2.0).
```